api/src/main/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminServiceImpl.java

```diff
@@ -392,7 +392,6 @@ public DbWorkspace setPublished(
         publish ? WorkspaceAccessLevel.READER : WorkspaceAccessLevel.NO_ACCESS;
 
     // Enable all users in RT or higher tiers to see all published workspaces regardless of tier
-    // Note: keep in sync with WorkspaceAuthService.updateWorkspaceAcls.
```

Comment:
no longer true.  This was added in https://github.com/all-of-us/workbench/pull/6464 because of old logic where we needed to replace the whole ACL when sharing.  Now we use patch semantics, so existing ACL entries are unaffected, and there is no logic to keep in sync.